### PR TITLE
Add Task 224

### PR DESCRIPTION
**Pull request by vladklopotenko**

### CODE REVIEW

File: Projects/TEAM-2/Klopotenko/224/224/Program.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+using System;
+
+namespace _224
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] array = new int[10];
+            Console.WriteLine("Complete the array with 10 integer values from -20 to 20. Entrr \"Stop\" when you want finish: \n");
```
Comment (LoveMyDaughter) [inline Projects/TEAM-2/Klopotenko/224/224/Program.cs:17]: "Stop" doesn't work...

**Review by LoveMyDaughter — COMMENTED** [review]
I'm not sure you've understood the task. I think you should first take your initial array, find the leftmost and the rightmost negatives and then sort only elements between negative numbers.
For example: Entered array is: 2 0 7 -9 3 -1 6 -5 7 2
Result should be: 2 0 7 -9 -1 3 6 -5 7 2
Your result is: -9 -5 0 2 2 3 6 7 7 -1

File: Projects/TEAM-2/Klopotenko/224/224/Program.cs (hunk 1 above)
Comment (vladklopotenko) [reply Projects/TEAM-2/Klopotenko/224/224/Program.cs:17]: I removed "Stop" from project

Comment (vladklopotenko) [reply Projects/TEAM-2/Klopotenko/224/224/Program.cs:17]: I understood this task that I need to find the smallest and the biggest negative numbers and sort all numbers from array between them. I will try to remake this task

Comment (LoveMyDaughter) [reply Projects/TEAM-2/Klopotenko/224/224/Program.cs]: Keep it. I'm not sure that I'm right about the task.

Comment (vladklopotenko) [reply Projects/TEAM-2/Klopotenko/224/224/Program.cs]: Ok

**Review by LoveMyDaughter — APPROVED** [review]

**Pull request closed without merging**